core/src/apps/cardano/addresses.py

```diff
@@ -231,6 +233,16 @@ def validate_output_address_parameters(
     assert_params_cond(parameters.address_type in ADDRESS_TYPES_PAYMENT_KEY)
 
 
+def validate_governance_reward_address_parameters(
+    parameters: messages.CardanoAddressParametersType,
+) -> None:
+    validate_address_parameters(parameters)
```

Comment:
In `validate_governance_reward_address`, we check `network_id`, but here we don't? (Not sure.)

Reply:
Yes (the same is true for output addresses): Address parameters don't contain the network id - we just use the network id from the tx global context when deriving the address string. On the other hand, `validate_governance_reward_address` needs to check whether the global network id matches the network serialized in the address string.